models/User.go

```diff
@@ -1801,6 +1808,10 @@ func PerformOrderFillTransaction(ask *Ask, bid *Bid, stockTradePrice uint64, sto
 	askTransaction := GetTransactionRef(ask.UserId, ask.StockId, OrderFillTransaction, 0, stockTradePrice, total)
 	bidTransaction := GetTransactionRef(bid.UserId, bid.StockId, OrderFillTransaction, int64(stockTradeQty), stockTradePrice, -total+reservedCashForTrade)
 
+	//Transaction to update cash used for Bid Order and Stocks reserved for Ask Order
+	askUpdateTransaction := GetTransactionRef(ask.UserId, ask.StockId, ReserveUpdateTransaction, int64(stockTradeQty), stockTradePrice, 0)
```

Comment:
Same here and the line below. Ideally, you shouldn't need comments. The way you name your functions, variables etc should explain what your code is trying to achieve.